### PR TITLE
Suggested fix for bug#19356.

### PR DESCRIPTION
**Pull request by ebezault**
If it looks good to you, then I'll try to commit it to SVN.

### CODE REVIEW

File: Src/C/run-time/eif_threads.h [hunk 1]
```diff
@@ -66,10 +66,10 @@
 #define eif_thr_default_priority() 0
 #define eif_thr_min_priority() 0
 #define eif_thr_max_priority() 0
-#define eif_thr_create_with_attr(current_obj, init_func, attr)
-#define eif_thr_create_with_attr_new(current_obj, init_func, log_id, is_proc, attr)
+#define eif_thr_create_with_attr(current_obj, init_func, set_terminated_func, attr)
+#define eif_thr_create_with_attr_new(current_obj, init_func, set_terminated_func, log_id, is_proc, attr)
 /* Obsolete `eif_thr_create_with_args' but maintained to make it easy to switch between runtime versions. */
-#define eif_thr_create_with_args(current_obj, init_func, priority, policy, detach)
+#define eif_thr_create_with_args(current_obj, init_func, set_terminated_func, priority, policy, detach)
```
Comment (ebezault) [inline Src/C/run-time/eif_threads.h:72]: Is this function still needed? Or is it now old enough that we can get rid of it?

Comment (manuseiffel) [reply Src/C/run-time/eif_threads.h:72]: No it is not except a reference in Gobo. So we could remove it in a separate commit.

**Comment by manuseiffel** [comment]
LGTM except you should not commit anything in experimental. For the time being we merge the code on a regular basis to the experimental version.

File: Src/C/run-time/eif_threads.h (hunk 1 above)
Comment (ebezault) [reply Src/C/run-time/eif_threads.h:72]: I see no reference to it in Gobo. Where did you see it?

**Comment by ebezault** [comment]
OK for experimental.
Committed in SVN in rev#100327.

File: Src/C/run-time/eif_threads.h (hunk 1 above)
Comment (ebezault) [reply Src/C/run-time/eif_threads.h:72]: I removed `eif_thr_create_with_args' in rev#100328.

Comment (manuseiffel) [reply Src/C/run-time/eif_threads.h:72]: Sorry, it was a trace of its former presence in the old gobo, not the new one.

**Pull request closed without merging**